pkgs/top-level/all-packages.nix

```diff
@@ -31280,6 +31280,10 @@ with pkgs;
 
   octoprint = callPackage ../applications/misc/octoprint { };
 
+  oculante = callPackage ../applications/graphics/oculante {
+    inherit (darwin.apple_sdk.frameworks) CoreFoundation;
```

Comment:
Please don't inherit in all-packages.nix, use `darwin` directly instead, also this is unused